runtime/compilers/rillv1/util.go

```diff
@@ -0,0 +1,29 @@
+package rillv1
+
+import (
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+)
+
+type ProjectConfig struct {
+	// Project variables
+	Variables   map[string]string `yaml:"env,omitempty"`
+	Title       string            `yaml:"title,omitempty"`
+	Description string            `yaml:"description,omitempty"`
+}
+
+func HasRillProject(dir string) bool {
+	_, err := os.Open(filepath.Join(dir, "rill.yaml"))
+	return err == nil
+}
+
+func ParseProjectConfig(content []byte) (*ProjectConfig, error) {
+	c := &ProjectConfig{Variables: make(map[string]string)}
+	if err := yaml.Unmarshal(content, c); err != nil {
+		return nil, err
+	}
+
+	return c, nil
+}
```

Comment:
We should avoid having this `ProjectConfig` type in this package, it will be confusing vs. the existing functionality here. The caller should either be refactored to use `ParseRillYAML` or since it's only one place and only seems to use the `Title` and `Description` fields, it *might* make sense to just inline that parsing functionality there.